src/main/java/io/github/cottonmc/cotton/gui/widget/WButton.java

```diff
@@ -92,21 +104,21 @@ public void paint(MatrixStack matrices, int x, int y, int mouseX, int mouseY) {
 		}
 		
 		float px = 1/256f;
-		float buttonLeft = 0 * px;
-		float buttonTop = (46 + (state*20)) * px;
+		float pxButtonLeft = 0 * px;
+		float pxButtonTop = (46 + (state*getHeight())) * px;
 		int halfWidth = getWidth()/2;
 		if (halfWidth>198) halfWidth=198;
-		float buttonWidth = halfWidth*px;
-		float buttonHeight = 20*px;
+		float pxButtonWidth = halfWidth*px;
+		float pxButtonHeight = getHeight()*px;
 		
 		float buttonEndLeft = (200-(getWidth()/2)) * px;
 
 		Identifier texture = getTexture();
-		ScreenDrawing.texturedRect(matrices, x, y, getWidth()/2, 20, texture, buttonLeft, buttonTop, buttonLeft+buttonWidth, buttonTop+buttonHeight, 0xFFFFFFFF);
-		ScreenDrawing.texturedRect(matrices, x+(getWidth()/2), y, getWidth()/2, 20, texture, buttonEndLeft, buttonTop, 200*px, buttonTop+buttonHeight, 0xFFFFFFFF);
+		ScreenDrawing.texturedRect(matrices, x, y, getWidth()/2, getHeight(), texture, pxButtonLeft, pxButtonTop, pxButtonLeft+pxButtonWidth, pxButtonTop+pxButtonHeight, 0xFFFFFFFF);
+		ScreenDrawing.texturedRect(matrices, x+(getWidth()/2), y, getWidth()/2, getHeight(), texture, buttonEndLeft, pxButtonTop, 200*px, pxButtonTop+pxButtonHeight, 0xFFFFFFFF);
```

Comment:
These changes are not useful since they're only variable renames. `WButton`'s height is constant so it can remain 20 in this method as well.